test/plugin/scenarios/mysql-scenario/src/main/java/org/apache/skywalking/apm/testcase/mysql/SQLExecutor.java

```diff
@@ -53,28 +54,28 @@ public void insertData(String sql, String id, String value) throws SQLException
     public void dropTable(String sql) throws SQLException {
         executeStatement(sql);
     }
-    
+
```

Comment:
This seems unexpected change?

Reply:
it seems some Newline character different